driver/pace/driver/driver.py

```diff
@@ -122,6 +123,7 @@ class DriverConfig:
     pair_debug: bool = False
     output_initial_state: bool = False
     output_frequency: int = 1
+    safety_check_frequency: int = 1
```

Comment:
I think by default we should not do safety check, I would suggest we set this to a very large number, or we should even have a switch to turn on/off?